### ISSUE TITLE
Support OneToMany relations

### ISSUE THREAD

**Issue by pschaub** [E-vence/SoftDeleteableListenerExtensionBundle#18]
Currently ManyToOne, ManyToMany and OneToOne are supported.

**Annotation on OneToMany relations**
Currently the annotation is ignored if you use it on an OneToMany relation.

**Workaround**
You need to set the annotation to the mapped side (ManyToOne) instead of the inversed side (OneToMany).

**Suggestion**
There should be support for "OneToMany" relations.
OR
There should be a error message to warn the developer about the mistake.

**Comment by RubenHarms** [comment]
I agree to you! #19 was a recent improvement for the inversed side of the ManytoMany. The code should be similar for the inversed side of ManyToOne (OneToMany). @RuSPanzer  isn't it?

**Comment by RuSPanzer** [comment]
If OneToMany and ManyToOne bidirectional relations, just mark onSoftDelete in mapped side (entity with ManyToOne annotation).
If reference unidirectional... Seems to need to be removed all dependent entities with $collection->clear().  it's more like mapped side MTM.
